lib/docker-compose/models/compose_container.rb

```diff
@@ -164,6 +166,14 @@ def prepare_environment(env_entries)
   end
 
   #
+  # Forces the labels structure to use the hash format.
+  #
+  def prepare_labels(labels)
```

Comment:
You're right.
I'll check how exactly compose label its containers and try to implement the same heuristic in this project.
For now, we can leave the implementation as it is (no random label assigned).

Thanks!